### PR TITLE
docs: add website setup to readme

### PR DESCRIPTION
**Pull request by rukundob451**
**Description**

- steps of hosting modelina website locally
- contributing to modelina

**Related issue(s)**

Fixes https://github.com/asyncapi/modelina/issues/1299

### CODE REVIEW

File: modelina-website/README.md [hunk 1]
```diff
@@ -16,4 +16,39 @@ Here is a quick overview of where some of the functions for rendering the playgr
...
+1. Fork the repository by clicking on `Fork` option on top right of the main repository.
+
+2. Open Command Prompt on your local computer.
+
+3. Clone the forked repository by adding your own GitHub username in place of `<username>`.
+   For multiple contributions it is recommended to have [proper configuration of forked repo](https://github.com/asyncapi/community/blob/master/git-workflow.md).
+
+```bash
+    git clone https://github.com/<username>/website/
+```
```
Comment (jonaslagoni) [inline modelina-website/README.md:32]: I think we should just link to https://github.com/asyncapi/community/blob/master/git-workflow.md and that's it, at least for the first 3 points 🙂 

No need to repeat what it say.

**Review by jonaslagoni — CHANGES_REQUESTED** [review]
Just have one comment about the git setup 🙂 

For the PR title make sure you use `docs:` instead of `feat:` we don't want to release a new version of Modelina for documentation changes 🙂

**Comment by rukundob451** [comment]
> Just have one comment about the git setup 🙂
> 
> For the PR title make sure you use `docs:` instead of `feat:` we don't want to release a new version of Modelina for documentation changes 🙂

Thank you for the suggestion @jonaslagoni, I don't know how I forgot to put that. Will do so in my next PR.

File: modelina-website/README.md (hunk 1 above)
Comment (rukundob451) [reply modelina-website/README.md:32]: Ohhhh, I get you @jonaslagoni. Let's do that indeed no need to repeat we shall just link to https://github.com/asyncapi/community/blob/master/git-workflow.md

**Comment by rukundob451** [comment]
@jonaslagoni, I have changed as suggested [commit](https://github.com/asyncapi/modelina/pull/1395/commits/870b63ea0b2265ee1a4f1da13b077ecab445727b)

File: modelina-website/README.md [hunk 2]
```diff
@@ -16,4 +16,32 @@ Here is a quick overview of where some of the functions for rendering the playgr
...
+    npm install
+```
+
+3. Run the website locally.
+
+```bash
+    npm run dev
+```
+
+4. ccess the live development server at [localhost:3000](http://localhost:3000).
```
Comment (sambhavgupta0705) [inline modelina-website/README.md:45]:
```suggestion
4. Access the live development server at [localhost:3000](http://localhost:3000).
```

File: modelina-website/README.md [hunk 3]
```diff
@@ -16,4 +16,32 @@ Here is a quick overview of where some of the functions for rendering the playgr
...
+## Run modelina locally
+
+Contributions to `modelina` check out [git-workflow](https://github.com/asyncapi/community/blob/master/git-workflow.md)
+
+
+
+1. Navigate to the website directory.
+
+```bash
+    cd website
```
Comment (sambhavgupta0705) [inline modelina-website/README.md:30]:
```suggestion
    cd modelina-website
```

File: modelina-website/README.md [hunk 4]
```diff
@@ -16,4 +16,32 @@ Here is a quick overview of where some of the functions for rendering the playgr
 - `src/components/playground/PlaygroundOptions.tsx` is the main component that renders the options based on which output is selected.
 - `src/components/playground/options` contain all the individual react components for showing the output options.
 - `src/components/playground/Playground.tsx` is the main playground component, and is the one rendered by the playground page.
-- `src/components/playground/GeneratedModels.tsx` is the playground component responsible for rendering the generated models.
\ No newline at end of file
+- `src/components/playground/GeneratedModels.tsx` is the playground component responsible for rendering the generated models.
+
+## Run modelina locally
```
Comment (sambhavgupta0705) [inline modelina-website/README.md:21]:
```suggestion
## Run Modelina Website locally
```

File: modelina-website/README.md [hunk 5]
```diff
@@ -16,4 +16,32 @@ Here is a quick overview of where some of the functions for rendering the playgr
...
+
+Contributions to `modelina` check out [git-workflow](https://github.com/asyncapi/community/blob/master/git-workflow.md)
```
Comment (sambhavgupta0705) [inline modelina-website/README.md:23]:
```suggestion
Checkout documentation that explains [git-workflow](https://github.com/asyncapi/community/blob/master/git-workflow.md) used in our repositories. 
```

**Review by sambhavgupta0705 — CHANGES_REQUESTED** [review]

**Comment by sambhavgupta0705** [comment]
@rukundob451 found some errors, kindly reslove them.

**Comment by sambhavgupta0705** [comment]
@rukundob451 take a look of this comment of Jonas 

> But agree we should probably add a section about the website in the main readme

**Comment by rukundob451** [comment]
> @rukundob451 found some errors, kindly resolve them.

Looking at them now

**Comment by rukundob451** [comment]
> @rukundob451 take a look of this comment of Jonas
> 
> > But agree we should probably add a section about the website in the main readme

checking out now

**Comment by rukundob451** [comment]
> Just have one comment about the git setup 🙂
> 
> For the PR title make sure you use `docs:` instead of `feat:` we don't want to release a new version of Modelina for documentation changes 🙂

Yes, I have taken note of this and made the necessary change as.

File: modelina-website/README.md (hunk 4 above)
Comment (rukundob451) [reply modelina-website/README.md:21]: I have done this,

File: modelina-website/README.md (hunk 5 above)
Comment (rukundob451) [reply modelina-website/README.md:23]: I have resolved this.

File: modelina-website/README.md (hunk 3 above)
Comment (rukundob451) [reply modelina-website/README.md:30]: Resolved this

File: modelina-website/README.md (hunk 2 above)
Comment (rukundob451) [reply modelina-website/README.md:45]: Thanks for this clarification

**Comment by rukundob451** [comment]
@sambhavgupta0705 and @jonaslagoni, I have done the changes as suggested.

**Review by jonaslagoni — APPROVED** [review]
LGTM 👍

File: modelina-website/README.md [hunk 6]
```diff
@@ -16,4 +16,32 @@ Here is a quick overview of where some of the functions for rendering the playgr
...
+## Run Modelina Website locally
+
+Checkout documentation that explains [git-workflow](https://github.com/asyncapi/community/blob/master/git-workflow.md) used in our repositories.
+
+
+
+1. Navigate to the website directory.
+
+```bash
+    cd website-website
```
Comment (sambhavgupta0705) [inline modelina-website/README.md:30]: @jonaslagoni @rukundob451 there is a still an error here 😓 
cd modelina-website

Comment (rukundob451) [reply modelina-website/README.md:30]: I have checked this out now and have resolved it, thank you for the clarification.

**Pull request merged**